hathor/pubsub.py

```diff
@@ -90,6 +124,14 @@ class HathorEvents(Enum):
 
     CONSENSUS_TX_UPDATE = 'consensus:tx_update'
 
+    NETWORK_NEW_TRANSACTION_ACCEPTED = 'network:new_transaction_accepted'
```

Comment:
Is this just an alias for `NETWORK_NEW_TX_ACCEPTED`, or does the latter also include blocks?

Reply:
Oh, this is definitely a merge artifact that I missed.

Reply:
Fixed.